src/main/java/command/Command.java

```diff
@@ -0,0 +1,56 @@
+package command;
+
+/**
+ * Enumerates the types of possible commands.
+ */
+public enum Command {
+    EXIT_CMD    ("bye", true),
+    LIST_CMD    ("list", true),
+    DONE_CMD    ("done", false),
+    TODO_CMD    ("todo", false),
+    EVENT_CMD   ("event", false),
+    DEADLINE_CMD("deadline", false),
+    DELETE_CMD  ("delete", false),
+    FIND_CMD    ("find", false);
+
+    private final String cmdString;
+    private final boolean noArgs;
```

Comment:
Maybe you could rename 'noArgs' to 'hasNoArguments' or 'hasArguments' <--[you might have to rewrite some code if you change to this :(]

Reply:
Yup, I agree! I think 'hasNoArgs' may be easier to understand than 'isNoArgs' :)

Reply:
Fixed, thanks!